frappe/workflow/doctype/workflow_action/workflow_action.py

```diff
@@ -329,7 +329,7 @@ def get_users_next_action_data(transitions, doc):
 	def user_has_permission(user: str) -> bool:
 		from frappe.permissions import has_permission
 
-		return has_permission(doctype=doc, user=user, print_logs=False)
```

Comment:
This is valid in `develop`, it's likely breaking in v15 because it's not valid there.